spec/spec_helper.rb

```diff
@@ -0,0 +1,19 @@
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.